docs/FAQ/DTP.md

```diff
@@ -0,0 +1,137 @@
+
+> Q: What can I do with DP?
+>
+> A: You can buy stuff in the store with DP or buy [XP%](#xpeff) with DP. Additionally, DP is required for trading as a form of tax.
+
+### XP & Levels
+
+> Q: How can I level up?
+>
+> A: You can level up by getting XP.
+
+> Q: What happens when I level up?
+>
+> A: You get lootboxes. These lootboxes are different from vote crates, it may contain items rarer than items contained in vote crates.
+
+> Q: How can I get XP?
+>
+> A: You can get XP by killing people in [PvP](ServernGamemodes.md#PvP), killing bosses or mobs in [PvE](ServernGamemodes.md#PvE), surviving rounds in [Zombies](ServernGamemodes.md#Zombies)(You can play [hardcore zombies](ServernGamemodes.md#Hardcore-Zombies) to get XP faster) or [voting](https://terraria-servers.com/server/90/vote/) for the server, voting gives you 25% of the remaining XP you need to level up.
+
+<a name="xpeff"></a>
+
+> Q: How can I buy XP%?
+>
+> A: You can buy XP% by using the bar near the Get Crew button(or the Crew Badge if you're crew), (As of Oct 10 2019)each XP% costs 10 DP. XP% Can go past 100% leading in more XP. 
+
+> Q: How does XP% work?
+>
+> A: (As of Oct 10 2019)The XP formula is: `XP * XP%`.
+>
+> Say I have 50% XP% and I receive 500XP, the server will multiply `500 by 50%(or 0.5)` which means I only get half of it(250 XP), this formula applies no matter how much XP% whether it is 50% or 1000%.
+
+### Crew
+
+> Q: What is Crew?
+> 
+> A: Crew is a free set of permissions available to anyone **WHO OWNS** on Steam the game. You can use it to build in [Build](ServernGamemodes.md#Build), TP to anyone(Bannable if used in [PvP](ServernGamemodes.md#PvP)), summon bosses, or create houses in Survival
+
+> Q: How can I get Crew?
+>
+> A: Login on [DTP](#Trading-System) then simply click on the get crew button and proceed to verify you bought the game. Note that you need to have your games set to public in Steam(`Account Info-> Privacy Settings -> My Profile > Public -> Game Details > Public`) for this to work
+
+> Q: I don't own Terraria on Steam, is there anything else I can do to get Crew?
+>
+> A: An alternate way to get crew is by buying it with credits, 2 credits are required.
+>
+> Each vote gives 0.05 credits, that's about 40 days of voting.
+
+> Q: I have Crew but I can't use any/some of the features of Crew mentioned above.
+>
+> A: You can fix this by logging in at [DTP](#Trading-System)(Don't close the tab) then going [here](https://t.dark-gaming.com/fixcrew).
+
+### Credits
+
+> Q: What are Credits?
+>
+> A: Credits are basically the Platinum Coins of Dark Gaming. Except you can't convert DP to credits.
+
+> Q: How can I get Credits?
+>
+> A: Credits can be obtained by [voting](https://terraria-servers.com/server/90/vote/) for DG. Each vote gives you 0.05 Credits.
+
+> Q: What can I do with Credits?
+>
+> A: You can craft custom Titles or Tags using credits, you can also get Crew using 2 Credits.
+
+### Crafting
+**UNAVAILABLE!**
```

Comment:
Unavailable?

Reply:
It's not shown in the DTP menu

Reply:
It is for me, do you have enough credits to use it?

Reply:
Oh, I didn't think it'd only show up when you have enough credits, I have enough credits but didn't notice.